lib/suspenders/app_builder.rb

```diff
@@ -401,6 +401,10 @@ def setup_bundler_audit
       append_file "Rakefile", %{\ntask default: "bundler:audit"\n}
     end
 
+    def setup_spring
+      bundle_command 'exec spring binstub --all'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.